apps/condo/domains/billing/schema/SumBillingReceiptsService.js

```diff
@@ -4,72 +4,107 @@
 
 const Big = require('big.js')
 
-const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
-const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+const { getDatabaseAdapter } = require('@open-condo/keystone/databaseAdapters/utils')
+const { GQLError } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, getSchemaCtx, allItemsQueryByChunks } = require('@open-condo/keystone/schema')
 
 const access = require('@condo/domains/billing/access/SumBillingReceiptsService')
+const { PERIOD_REGEX } = require('@condo/domains/billing/constants/constants')
 const { WRONG_VALUE } = require('@condo/domains/common/constants/errors')
-const { GqlWithKnexLoadList } = require('@condo/domains/common/utils/serverSchema')
+
 
 const ERRORS = {
-    PERIOD_BADLY_SPECIFIED: {
-        mutation: 'registerBillingReceipts',
-        variable: ['where', 'period'],
-        code: BAD_USER_INPUT,
+    TIN_OR_ORGANIZATION_ID_MUST_BE_SPECIFIED: {
+        mutation: '_allBillingReceiptsSum',
+        variable: ['data'],
+        code: 'BAD_USER_INPUT',
+        type: WRONG_VALUE,
+        message: 'You must specify one of two values: tin or organizationId',
+    },
+    BAD_PERIOD_FORMAT: {
+        mutation: '_allBillingReceiptsSum',
+        variable: ['data', 'period'],
+        code: 'BAD_USER_INPUT',
         type: WRONG_VALUE,
-        message: 'You must specify period field for receipts in where query. where.period should be a simple string, example: _allBillingReceiptsSum(period: "2022-12-01")',
+        message: 'Bad period format, must be YYYY-MM-01. Example: 2022-02-01',
     },
 }
 
 const SumBillingReceiptsService = new GQLCustomSchema('SumBillingReceiptsService', {
-
     types: [
+        {
+            access: true,
+            type: 'input BillingReceiptsSumInput { period: String!, organizationId: ID, tin: String, importRemoteSystem: String }',
```

Comment:
`organizationId: ID` -- you need to use `organization: OrganizationWhereUniqInput`

Reply:
Is it possible to keep only `organization` argument? The arg 'tin' seems redundant.

Reply:
> Is it possible to keep only `organization` argument? The arg 'tin' seems redundant.

No, search by tin in this query used in `condo-restapi` service